src/xlsx.rs

```diff
@@ -332,8 +368,14 @@ impl<RS: Read + Seek> Xlsx<RS> {
                             _ => (),
                         }
                     }
-                    self.metadata.sheets.push(name.to_string());
-                    self.sheets.push((name, path));
+                    if !state.is_very_hidden() {
+                        self.metadata.sheets.push(name.to_string());
+                        self.sheets.push((name, path));
+                    } else if !name.is_empty() {
+                        debug!("Ignoring {name} because it's state is set to veryHidden");
+                    } else {
+                        debug!("Ignoring an unnamed sheet that has veryHidden state");
```

Comment:
can you remove these?

Reply:
Wait why do you want to remove them? Since it's using the tracing module it won't show up unless the log level is set to debug.

Reply:
idk, if it's not useful enough to set to info, i prolly don't want it in my dev environment unless im actively working on it in which case i would prolly add something in there with the specific thing i was looking at

Reply:
these aren't hardly ever going to get hit, so it prolly doesn't matter. i just don't like leaving random debug prints in

Reply:
ah I see the reasoning. So the thing is that it will be nearly impossible to debug without the logs since this state can only be set programmatically apparently. It wont show up in any editors, excel or sheets, outside of some visual basic editor. 

I think that I am going to leave it as is, since the visibility of the log is determined at runtime. Without the logs here, it would require recompiling and deploying downstream code.